spec.emu

```diff
@@ -262,9 +291,13 @@ location: https://github.com/tc39/proposal-decimal/
       <h1>Decimal128 ( _x_ [ , _options_ ] )</h1>
       <emu-alg>
         1. If NewTarget is *undefined*, throw a *TypeError* exception.
-        1. Let _n_ be ? ToDecimal128(_x_).
+        1. Let _toParse_ be *undefined*.
+        1. If _x_ is a BigInt, set _toParse_ to BigInt::toString(_x_, 10).
+        1. If _x_ is a Number, set _toParse_ to Number::toString(_x_, 10).
+        1. If _x_ is a String, set _toParse_ to _x_.
+        1. If _toParse_ is not a String, throw a *TypeError* exception.
```

Comment:
Nit: structure this as "If ..., Otherwise, if ... Otherwise"

Reply:
done